demo-project/src/demo_project/requirements.in

```diff
@@ -5,7 +5,6 @@ isort~=5.0
 jupyter~=1.0
 jupyter_client>=5.1, <7.0
 jupyterlab~=3.0
-kedro~=0.18.0
```

Comment:
why did we remove this?

Reply:
@noklam suggested it so it always pull the latest kedro version, rather than any specific version

Reply:
but there's no kedro dependency now. Should we instead do kedro<1.0 ??

Reply:
I am not sure if we use this anywhere in our tests. We only use - test_requirements.txt, docker_requirements.txt files for dependencies. The requirements.in is internal to demo-project I guess. I think for this PR, this change seems not relevant. Is this blocking the Kedro-Viz build in anyway ?

Reply:
Kedro-datasets on Kedro>0.19 so the existing pin is incorrect and resulted in conflict dependencies.

Does the latest viz still support older version?

Reply:
i dont think this blocking the build if we remove or update it to kedro >0.19. I'll put it back to with the latest version for kedro 👍

Reply:
Yes, Kedro-viz support Kedro 18. Data preview doesn't work as it's part of new datasets